streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBTimestampedStore.java

```diff
@@ -207,6 +207,15 @@ public void prepareBatch(final List<KeyValue<Bytes, byte[]>> entries,
             return new RocksDBDualCFIterator(name, innerIterWithTimestamp, innerIterNoTimestamp);
         }
 
+        /**
+         * @throws UnsupportedOperationException
+         */
+        @Override
+        public KeyValueIterator<Bytes, byte[]> prefix(Bytes prefix) {
+            //TODO - Bellemare - How do we support this here?
+            throw new UnsupportedOperationException("prefixScan() not supported in " + getClass().getName());
```

Comment:
I'm not really sure how the Timestamped statestores work, nor if I even need to support this (since my intention is to only use the non-timestamped versions).